engine/gamesys/src/gamesys/gamesys_private.cpp

```diff
@@ -337,6 +337,11 @@ namespace dmGameSystem
             info.m_StepFunction     = material_attributes[i].m_StepFunction;
             info.m_ValueVectorType  = material_attributes[i].m_VectorType;
 
+            if (info.m_CoordinateSpace == dmGraphics::COORDINATE_SPACE_DEFAULT)
+            {
+                info.m_CoordinateSpace = default_coordinate_space;
+            }
```

Comment:
Components that use the "vertex space" option in the materials (basically only models) will produce certain streams based on that value, i.e positions normals and tangents.